provider/k8s.go

```diff
@@ -382,6 +382,7 @@ type FileStore interface {
 	Upload(sourcePath string, destPath string) error
 	Download(sourcePath string, destPath string) error
 	FilestoreType() pc.FileStoreType
+	AddEnvVars(envVars map[string]string) map[string]string
```

Comment:
All of the things above no longer support this interface since your mixing pointer receivers with struct receivers